freqtrade/optimize/optimize_reports.py

```diff
@@ -662,7 +662,10 @@ def show_backtest_results(config: Dict, backtest_stats: Dict):
     if len(backtest_stats['strategy']) > 1:
         # Print Strategy summary table
 
+        first_strat_stats = next(iter(backtest_stats['strategy'].items()))[1]
```

Comment:
I think this should be simplified to simply (reuse) `results` from the above loop.

In python - the variable in a for loop will keep the value of the last loop. 
It will not have the first strategy results but the last ones - but i don't think that'll be a problem.

Reply:
you python people and your fancy useful shortcuts :) thanks. changed it that way